compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.td

```diff
@@ -412,10 +412,44 @@ def AMDAIEPackAndTranspose :
   ];
 }
 
-def AMDAIEPackToDma :
-  Pass<"iree-amdaie-pack-to-dma", ""> {
-  let summary = "Convert pack/unpack ops to AMDAIE DMA ops operating on logical objectFifos.";
-  let constructor = "mlir::iree_compiler::AMDAIE::createAMDAIEPackToDmaPass()";
+def AMDAIEConvertToDma :
+  Pass<"iree-amdaie-convert-to-dma", ""> {
+  let summary = "Convert linalg ops to AMDAIE DMA ops on logical objectFifos.";
+  let description = [{
+   Converts/lowers linalg.copy, linalg_ext.pack, and linalg_ext.unpack ops to
+   dma operations. For example, given allocactions,
+
+    ```mlir
+    %src = memref.alloc() : memref<8x16xi32, 1>
+    %dst = memref.alloc() : memref<1x1x8x16xi32, 1>
+    ```
+
+    a packing copy operation between them,
+    ```mlir
+    iree_linalg_ext.pack %src inner_dims_pos = [0, 1] inner_tiles = [8, 16]
+                    into %dst : (memref<8x16xi32, 1> memref<1x1x8x16xi32, 1>)
+    ```
+
+    gets lowered to,
+    ```mlir
+    %0 = amdaie.logicalobjectfifo.from_memref %dst, {} : memref<1x1x8x16xi32, 1>
+                              -> !amdaie.logicalobjectfifo<memref<1x1x8x16xi32, 1>>
+    %1 = amdaie.logicalobjectfifo.from_memref %src, {} : memref<8x16xi32, 1>
+                                -> !amdaie.logicalobjectfifo<memref<8x16xi32, 1>>
+    %2 = amdaie.dma_cpy_nd(%0[0, 0, 0, 0] [1, 1, 8, 16] [128, 128, 16, 1],
+                           %1[0, 0, 0, 0] [1, 1, 8, 16] [128, 16, 16, 1]) :
+                          (!amdaie.logicalobjectfifo<memref<1x1x8x16xi32, 1>>,
+                           !amdaie.logicalobjectfifo<memref<8x16xi32, 1>>)
+    ```
+
+    The current approach taken with linalg.copy operations, is to lower them
+    to identity linalg_ext.pack operations, and then rely on the lowering of
+    linalg_ext.pack to dma operations to handle the actual lowering to DMAs.
+    We want want to reconsider this 'indirection' in the future.
```

Comment:
Nit: use "iree_linalg_ext' and remove additional "want". In addition, I would delete the last sentence in this file and move it to where it is implemented in cpp.